src/assets/scripts/client/InputController.js

```diff
@@ -713,6 +714,16 @@ export default class InputController {
     }
 
     /**
+     * Disable default behavior of right-click menu appearing on right click
+     *
+     * @for InputController
+     * @method _disableRightClickMenu
+     */
+    _disableRightClickMenu() {
```

Comment:
@erikquinn why do we need this in addition to the `preventDefault` we already have?

his is a clever solution but I'm really not a fan.  We can solve this in a much simpler way.

Reply:
Tried the combinations of all options offered up above, both in right-click events' mouse-UP and mouse-DOWN events (since it seems to appear on mouse up anyway), but sadly got no results from them. :disappointed: